programmers/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,9 @@
+function solution(s) {
+    const Sleng = s.length;
+    const Center = parseInt(Sleng / 2);
+    if ( Sleng % 2 === 0) {
+        return s.substr(Center-1,2);
+    }
+    else 
+        return s.slice(Center,Center + 1);
+}
```

Comment:
1. 변수명 지을 때 카멜 표기법을 사용하는게 좋음.
    - 카멜 표기법(Camel Case)
  첫 글자를 대문자로 적되, 맨 앞에 오는 글자는 소문자로 표기하는 것이다.
  표기한 모습이 낙타의 등과 같다고 하여 카멜 표기법이라고 부른다. 
  ex) Camera + Button => cameraButton

2. s.length(길이)과 같은 상수의 명명법은  GNU Naming Convention 을 사용
  ex) DEFAULT_COUNTRY_CODE

3. if ... else문에서 안에서 return이 나오면 else를 안써도 그외의 경우에 return을 함

```suggestion
function solution(s) {
    const S_LENGTH = s.length;
    const center = parseInt(S_LENGTH / 2);
    
    //if문에서 조건식() 내부의 양끝은 공백x 연산자 양 끝 공백, 쉼표 이후 공백
    //한 줄로 짠다면 { } 제거 가능
    if (S_LENGTH % 2 === 0) return s.substr(center - 1, 2); 
    return s.slice(center, center + 1);
}
```